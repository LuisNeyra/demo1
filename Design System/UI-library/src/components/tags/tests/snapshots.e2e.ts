import { snapIt } from '../../../test';

const component = 'tags';

describe(component, () => {
  const snap = snapIt(component);

  describe('renders', () => {
    snap('renders correctly with no props', '<blaze-tags></blaze-tags>');
    snap('placeholder updates', '<blaze-tags placeholder="Test Placeholder"></blaze-tags>');
    snap(
      'options renders select dropdown',
      `<blaze-tags placeholder="Select a tag" options='[{ "text": "Test", "value": "test" }, { "text": "Test Two", "value": "test two" }, { "text": "Test Three", "value": "test three" }]'></blaze-tags>`
    );
  });
});
