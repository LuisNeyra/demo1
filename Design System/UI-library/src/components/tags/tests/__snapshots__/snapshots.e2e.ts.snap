// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tags renders options renders select dropdown 1`] = `"<blaze-tags placeholder=\\"Select a tag\\" options=\\"[{ &quot;text&quot;: &quot;Test&quot;, &quot;value&quot;: &quot;test&quot; }, { &quot;text&quot;: &quot;Test Two&quot;, &quot;value&quot;: &quot;test two&quot; }, { &quot;text&quot;: &quot;Test Three&quot;, &quot;value&quot;: &quot;test three&quot; }]\\" class=\\"hydrated\\"><div class=\\"c-tags\\"><div class=\\"c-tags__container\\"></div><div class=\\"c-tags__field-container\\"><select class=\\"c-field\\"><option>Select a tag</option><option value=\\"test\\">Test</option><option value=\\"test two\\">Test Two</option><option value=\\"test three\\">Test Three</option></select></div></div></blaze-tags>"`;

exports[`tags renders placeholder updates 1`] = `"<blaze-tags placeholder=\\"Test Placeholder\\" class=\\"hydrated\\"><div class=\\"c-tags\\"><div class=\\"c-tags__container\\"></div><div class=\\"c-tags__field-container\\"><input type=\\"text\\" class=\\"c-field\\" placeholder=\\"Test Placeholder\\"></div></div></blaze-tags>"`;

exports[`tags renders renders correctly with no props 1`] = `"<blaze-tags class=\\"hydrated\\"><div class=\\"c-tags\\"><div class=\\"c-tags__container\\"></div><div class=\\"c-tags__field-container\\"><input type=\\"text\\" class=\\"c-field\\" placeholder=\\"Add a tag\\"></div></div></blaze-tags>"`;
